cuecfg/cuecfg.go

```diff
@@ -52,7 +52,7 @@ func Unmarshal(data []byte, extension string, valuePtr any) error {
 			return errors.WithStack(err)
 		}
 		return nil
-	case ".xml", ".csproj":
+	case ".xml", ".csproj", ".fsproj":
```

Comment:
I don't love needing to hard code `.csproj` and `.fsproj` in this package. It is needed because the `cuecfg.ParseFiles` function derives the extension from the filename, and so there is no means of indicating what extension to use.

An alternative idea is to introduce `cuecfg.ParseFilesForExtension(filepath string, ext string)` and I can call it as `cuecfg.ParseFiles(projectPath, ".xml")`